### PR TITLE
GEODE-2854: GEODE REST API Docs Missing Authentication

### PR DESCRIPTION
**Pull request by davebarnes97**
Reviews requested from: @mmartell, @jaredjstewart @karensmolermiller @joeymcallister

### CODE REVIEW

File: geode-docs/rest_apps/setup_config.html.md.erb [hunk 1]
```diff
@@ -29,162 +31,233 @@ All Geode REST interface classes and required JAR files are distributed as a WAR
...
+-   Application peer member caches will access REST-accessible regions (resources) with `Region.get(key)`.
+-   Your deployment has persistent regions that must be available as resources to the REST API.  
 
-            ``` pre
-            gfsh>configure pdx --read-serialized=true --disk-store
-            ```
+To configure PDX in your cluster, perform the following steps:
 
-    After you have configured PDX for your caches, then proceed with starting up your REST-enabled servers and other data nodes.
+1.  Start up a locator running the [cluster configuration service](../configuring/cluster_config/gfsh_persist.html) (enabled by default). For example:
```
Comment (joeymcallister) [inline geode-docs/rest_apps/setup_config.html.md.erb]: Change "Start up a locator" to "Start a locator"

File: geode-docs/rest_apps/setup_config.html.md.erb [hunk 2]
```diff
@@ -29,162 +31,233 @@ All Geode REST interface classes and required JAR files are distributed as a WAR
...
 
     ``` pre
-    gfsh>start server --name=server1  --start-rest-api=true \
-    --http-service-port=8080 --http-service-bind-address=localhost \
-    --J=-Dgemfire.jmx-manager=true --J=-Dgemfire.jmx-manager-start=true
+    gfsh>configure pdx --read-serialized=true --disk-store
     ```
 
-    Note that when started as a JMX Manager, the server will also host the Pulse web application in the same HTTP service.
+    After you have configured PDX for your caches, then proceed with starting up your REST-enabled servers and other data nodes.
```
Comment (joeymcallister) [inline geode-docs/rest_apps/setup_config.html.md.erb]: Change "starting up your" to "starting your"

File: geode-docs/rest_apps/setup_config.html.md.erb [hunk 3]
```diff
@@ -29,162 +31,233 @@ All Geode REST interface classes and required JAR files are distributed as a WAR
...
+```
+http://super-user:1234567@localhost:8080/geode/v1
+```
 
-        If you did not specify these properties upon server startup or in `gemfire.properties`, then use the default of localhost and port 7070. See [Using the Swagger UI to Browse REST APIs](using_swagger.html#concept_rlr_y3c_54) for more information.
 
-## Programmatic Startup
+# <a id="setup_config_implementing_auth" class="no-quick-link"></a>Programmatic Startup
 
 You can also start up and configure Geode REST services programmatically. For example:
```
Comment (joeymcallister) [inline geode-docs/rest_apps/setup_config.html.md.erb]: Change "start up and" to "start and"

File: geode-docs/rest_apps/setup_config.html.md.erb [hunk 4]
```diff
@@ -29,162 +31,233 @@ All Geode REST interface classes and required JAR files are distributed as a WAR
...
+- [Implementing Authentication for the REST API](#setup_config_implementing_auth)
+- [Programmatic Startup](#setup_config_implementing_auth)
+
+# <a id="setup_config_enabling_rest" class="no-quick-link"></a>Enabling the REST API
+
+The REST API service for application development runs only on data nodes; you cannot run the service on a locator.
+
+To enable the Developer REST API service on a given server, set the `start-dev-rest-api` property
+to `true` when starting the data node to start an embedded Jetty server and deploy the Developer REST
+API WAR file on that node. Use either the `gfsh start server` command or the ServerLauncher API to enable this property.
```
Comment (karensmolermiller) [inline geode-docs/rest_apps/setup_config.html.md.erb]: To help me parse what is getting started and where with the property, can you add a comma or reword the sentence?

**Pull request closed without merging**